tests/e2e/data/validate_data_dir/invalid/networks-duplicate.yml

```diff
@@ -0,0 +1,30 @@
+os_migrate_version: 0.0.1
```

Comment:
We don't need any of the validation tests in e2e

Reply:
all removed